src/tests/thanks.test.ts

```diff
@@ -24,13 +24,13 @@ describe('thanks', () => {
           mentions: { users: new Map().set('86890631690977280', {}) }
         } as Partial<Message>) as never)
       ).toEqual(true));
-    test('returns true if thanking self with THANK YOU', () =>
+    test('returns false if thanking self with THANK YOU', () =>
```

Comment:
Ah I remembered why this is the case. There is another check in the file if the user is checking themselves, which will return a different message ("can't thank self").

Technically the regex change doesn't even know if the user is "thanking themselves", so this missing test-case is actually because the regex wont recognize `THANK YOU <name of user>` self or not.

Reply:
![image](https://user-images.githubusercontent.com/63889819/86413578-63845080-bc76-11ea-908c-58f185204762.png)

This may have been the case with gitter, but my code has a check to see if the mentioned user is the same as the message author.

Reply:
That check is somewhere else in our code:
https://github.com/bradtaniguchi/discord-bot-test/blob/40ab1a7dc56e05134dccb5b4c31660348430dfe8/src/commands/thanks.ts#L15

Reply:
Ah, I see.
So this test should return true just because the regex passes and the message contains a mention. 
Any idea why it doesn't return true?
I tested the regex and it does match...

Reply:
This is also fixed.